### DISCUSSION TITLE
能否增加数据库直接insert  update权限

### DISCUSSION THREAD

**Discussion by liuhaixiao** [hhyo/Archery#1833] [General]
我们这边想对特定的数据库 直接进行insert 或者update ，当然有这个数据库权限的用户 肯定是我们的技术总管级别的了  我们没有dba
不想走SQL审核，而且我们有很多私有化部署的 没法走sql审核

**Comment by liuhaixiao** [comment]
就是增加一个 这个用户 是不是可以对这个库直接insert或者update的权限

**Comment by LeoQuote** [comment]
可以配置自动通过审核， 不过这个方案对用户没有限制， 所有用户都可以提工单后自动通过并执行

**Comment by liuhaixiao** [comment]
我的意思是 能否在执行sql界面  用户可以针对某个可以insert的数据库直接insert  不走工单



---- 回复的原邮件 ----
| 发件人 | Leo ***@***.***> |
| 日期 | 2022年09月14日 18:22 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [hhyo/Archery] 能否增加数据库直接insert  update权限 (Discussion #1833) |

可以配置自动通过审核， 不过这个方案对用户没有限制， 所有用户都可以提工单后自动通过并执行

**Reply by LeoQuote** [reply]
这种需求应该不是archery的业务重点，你可以自己开发实现
